src/anomalib/engine/engine.py

```diff
@@ -390,7 +392,7 @@ def predict(
         CLI Usage:
             1. you can pick a model.
                 ```python
-                anomalib predict --model anomalib.models.Padim
+                anomalib predict --model Padim --data datasets/MVTec/bottle/test/broken_large
```

Comment:
this is how the example appears in the documentation:
![image](https://github.com/openvinotoolkit/anomalib/assets/17574420/57924e9c-1b80-463d-aed3-34fffb90512d)

Would it be en idea to use linebreaks for each arg for a better look?

```python
"""
        CLI Usage:
            1. you can pick a model.
                ```python
                anomalib predict --model anomalib.models.Padim
                anomalib predict --model Padim \
                                 --data datasets/MVTec/bottle/test/broken_large
                ```
            2. Of course, you can override the various values with commands.
                ```python
                anomalib predict --model anomalib.models.Padim \
                                 --data <CONFIG | CLASS_PATH_OR_NAME>
                ```
            4. If you have a ready configuration file, run it like this.
                ```python
                anomalib predict --config <config_file_path> --return_predictions
                ```
"""

```